lib/commands/changelog/index.js

```diff
@@ -0,0 +1,188 @@
+var progressBar = require("../../utils/progressBar");
+var execSync    = require("../../utils/execSync");
+var chalk       = require("chalk");
+
+var org  = process.env.GITHUB_ORG;
+var repo = process.env.GITHUB_REPO;
+
+var tags = [
+  "tag: breaking change",
+  "tag: spec compliancy",
+  "tag: new feature",
+  "tag: bug fix",
+  "tag: documentation",
+  "tag: internal",
+  "tag: polish"
+];
```

Comment:
Yeah, We are using this as a baseline for the babel changelog https://github.com/babel/babel/releases